### ISSUE TITLE
Cannot rename control scheme in Input Actions asset editor

### ISSUE THREAD

**Issue by YLlywelyn** [Unity-Technologies/InputSystem#554]
When editing an InputActions asset, whatever I type as the name for a new control scheme, it always has the name "New control scheme".  This happens when creating a new control scheme or editing an existing one.  Expected behaviour would be for the control scheme to be named what I named it (eg. "XInput" or "Keyboard").

**Comment by JiaUnity** [comment]
Not the case for me.
![RenameAction](https://user-images.githubusercontent.com/33637761/56448735-b9194b00-62c6-11e9-8259-aa3d72be861e.gif)
@JMSams What's the ISX package version and Unity version?

**Comment by YLlywelyn** [comment]
I can rename actions fine, it's when I try to name my control schemes I have issues.

Unity 2019.1.0f
Input system 0.2.6

**Comment by Rene-Damm** [comment]
This is a bug that has since been fixed on the `develop` branch. The problem is with focus. The renaming actually works but ONLY if you hit enter :) We were using a text field that only resulted in a change when you pressed enter OR focus moved away from the text field. For some reason I never quite pinpointed, the latter never worked. So you HAD to press enter or nothing happened.

On latest `develop` we switched away from this kind of text field. Should work as expected.

Fix should be in next package (due out this week).
